docs/en/compute-edition/32/rn/release-information/release-notes-32-00.adoc

```diff
@@ -43,23 +43,35 @@ Failure to upgrade Defenders will result in disconnection of any Defender versio
 [#upgrade-console]
 ==== Upgrade the Prisma Cloud Console
 
-With the 31.00.xxx release, the https://docs.paloaltonetworks.com/prisma/prisma-cloud/31/prisma-cloud-compute-edition-admin/welcome/support_lifecycle[supported Console versions (n, n-1, and n-2)] are v32.xx.xxx, v31.xx.xxx, and v30.xx.xxx.
+With the 32.00.xxx release, the https://docs.paloaltonetworks.com/prisma/prisma-cloud/31/prisma-cloud-compute-edition-admin/welcome/support_lifecycle[supported Console versions (n, n-1, and n-2)] are v32.xx.xxx, v31.xx.xxx, and v30.xx.xxx.
```

Comment:
The link needs to update to the new versioned docs for `v32` release: https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/....